src/tool/xmeta/cpp/models/namespace_model.cpp

```diff
@@ -0,0 +1,59 @@
+#include "namespace_model.h"
+
+namespace xlang
+{
+    namespace xmeta
+    {
+#pragma region namespace_body_model
+        bool namespace_body_model::member_id_exists(const std::string &member_id)
+        {
+            if (classes.find(member_id) != classes.end())
+                return true;
```

Comment:
Nit: The style convention from OSG is that you must have curly brackets and you cannot omit them here.